src/app/classes/diagram/diagram.ts

```diff
@@ -1,10 +1,17 @@
 import {Element} from './element';
+import { GraphTrace } from './GraphTrace';
 
 export class Diagram {
-    private readonly _elements: Array<Element>;
+    private readonly _elements: Array<Element>; // TODO entfernen?
```

Comment:
Von welchem Todo sprichst du hier oder ist nur der Kommentar versehentlich zurückgeblieben? :-)

Reply:
in den _elements im Diagram wären dann SVG Elemente die zu keinem Trace gehören.. Sowas gibt es gerade (noch?) nicht.. Ich entferne das mal noch